stream/harfanglab-intel/src/config.yml.sample

```
@@ -4,8 +4,8 @@ opencti:
 
 connector:
   id: 'ChangeMe'
-  type: 'INTERNAL_IMPORT_FILE'
-  name: 'Internal Import File Connector Template'
+  type: 'STREAM'
+  name: 'HarfangLab'
   scope: 'ChangeMe'
```

Good point ! 
We could also change the scope, as a general rule for a stream connector we use the connector name
```suggestion
  scope: 'harfanglab'
```